package.json

```diff
@@ -30,18 +30,18 @@
     "jquery-bar-rating": "^1.2.2"
   },
   "devDependencies": {
-    "@spryker/oryx": "^1.0.0",
-    "@spryker/oryx-for-zed": "^1.0.0",
+    "@spryker/oryx": "^1.2.0",
+    "@spryker/oryx-for-zed": "^1.0.0 || ^2.1.0",
```

Comment:
@danilmoro3 did you test this?

Reply:
@abelspry no, how we can test `oryx` ?

Reply:
ok, because oryx 1 is based on webpack 2, while oryx 2 is based on webpack 4 and architecture is different. Be sure that QA checks both because v2 might not work properly due to different architecture.